_docs/examples/windows.md

```diff
@@ -75,7 +76,8 @@ def main(target_process):
 """)
     script.on('message', on_message)
     script.load()
-    raw_input('[!] Press Enter at any time to detach from instrumented program.\n\n')
+    print("[!] Press Enter at any time to detach from instrumented program.\n\n")
```

Comment:
This message is no longer accurate, as EOF is now required (Ctrl+D on UNIX, Ctrl+Z on Windows/cmd.exe).